source/renderer/app/components/status/DaedalusDiagnostics.js

```diff
@@ -786,6 +793,16 @@ export default class DaedalusDiagnostics extends Component<Props, State> {
                     : intl.formatMessage(messages.statusOff)}
                 </td>
               </tr>
+              <tr>
+                <th>
+                  {intl.formatMessage(messages.systemTimeInTolerationMode)}:
+                </th>
+                <td className={this.getClass(!isSystemTimeInTolerationMode)}>
```

Comment:
Maybe this method `getClass` is not something that describes what we want, like we are getting class instance instead DOM class. Maybe something like `getStatusClassnames`?

Reply:
@tomothespian @daniloprates resolved in https://github.com/input-output-hk/daedalus/pull/1462/commits/0035e1196b8e87c9e99ad86155aa8775bc568afd